lua/grpc-hook.lua

```diff
@@ -62,6 +62,20 @@ local function init()
 
   function handler.onPlayerTrySendChat(playerID, msg, all)
     _G.GRPC.onChatMessage(playerID, msg, all)
+
+    local target = 0
+    if all then
+      target = 1
+    end
```

Comment:
Note: DCS events are always emitting `all=true` -- so this is a simulation related bug, unfortunately.

Reply:
I would say "report it to ED" but that would probably be a waste of time.